tests/test_BC.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr 23 11:34:50 2018
+
+@author: aero-10
+"""
+
+from IRCal.tools.blackbody_connect import BlackBody_Control as BC
+import pytest
+import os
+
+def test_write_file():
+    ''' Test to evaluate the write file function. 
+        Current functionality: test to make sure the files are the same
+        Future functionality: look for unicode errors and possibly some
+                              formatting
+    '''
+    filename = '/home/aero-10/Documents/SoftwareClass/IRCal/IRCal/tests/test_files/hello_world.c'
+    
+    def file_lines(filename):
```

Comment:
Function without a docstring. Running `pylint test_BC.py` will point stuff like this out for you.